lib/express-rate-limit.js

```diff
@@ -27,52 +53,41 @@ function RateLimit(options) {
         }
     });
 
+    options = defaults(options, {
```

Comment:
I needed a valid (default or not) `options.windowMs`, and that wasn't available until after the defaults were applied first on it.